network/meson.build

```diff
@@ -0,0 +1,11 @@
+install_data(
+    '01-network-manager-all.yml',
+    install_dir: sysconfdir / 'netplan'
+)
+
+# Override /usr/lib/NetworkManager/conf.d/10-globally-managed-devices.conf shipped by NetworkManager
+# See https://askubuntu.com/a/1075112
+install_data(
+    '10-globally-managed-devices.conf',
```

Comment:
I guess if it's the same file name the package will conflict right? So it would need to be a number with a higher priority?